tools/VscPreviewExtension/src/previewCore.ts

```diff
@@ -1,39 +1,35 @@
 // Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-import { Uri, ExtensionContext } from "vscode";
+import { ExtensionContext, window } from "vscode";
 import { MarkdownDocumentContentProvider } from "./markdownDocumentContentProvider";
-import { ChildProcessHost } from "./childProcessHost"
+import { ChildProcessHost } from "./childProcessHost";
+import * as ConstVariable from "./constVariable";
+import { DfmService } from "./dfmService";
 
-// Create a child process(DfmRender) by "_spawn" to render a html
 export class PreviewCore extends ChildProcessHost {
```

Comment:
Yes, it will be restructured in next PR.

---
In reply to: [111716417](https://github.com/dotnet/docfx/pull/1547#discussion_r111716417) [](ancestors = 111716417)